src/models/ministryPlate.ts

```diff
@@ -54,3 +60,92 @@ type Tyre = {
   PlyRating: string;
   FitmentCode: string;
 };
+
+export class MinistryPlateDocument extends DocumentModel {
+  constructor(request: Request) {
+    super(request.recipientEmailAddress);
+    const { vehicle, plate } = request;
+    const { techRecord } = vehicle;
+
+    this.documentType = DocumentName.MINISTRY;
+    this.filename = `plate_${request.plate.plateSerialNumber}`;
+
+    const plateData: Partial<MinistryPlate> = {
+      PlateSerialNumber: plate.plateSerialNumber,
+      DtpNumber: techRecord.brakes.dtpNumber,
+      PrimaryVrm: vehicle.primaryVrm,
+      Vin: vehicle.vin,
+      VariantNumber: techRecord.variantNumber,
+      ApprovalTypeNumber: techRecord.approvalTypeNumber,
```

Comment:
Should we not refactor this if we're doing it all?

Reply:
Good idea - means a small change to doc-gen which I can do as well  🙂